File: lib/bindings/tests/bindings/golang/test_breez_liquid_sdk.go
package main

import (
	"log"

	"example.org/golang/breez_liquid_sdk"
)

func main() {
	mnemonic := "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about"
	
	config := breez_liquid_sdk.DefaultConfig(breez_liquid_sdk.NetworkTestnet)

	sdk, err := breez_liquid_sdk.Connect(breez_liquid_sdk.ConnectRequest{
		Config: config,
		Mnemonic: mnemonic,
	})

	if err != nil {
		log.Fatalf("Connect failed: %#v", err)
	}

	info, err := sdk.GetInfo()

	if err != nil {
		log.Fatalf("GetInfo failed: %#v", err)
	}


	log.Print(info.Pubkey)
}

File: lib/bindings/tests/bindings/test_breez_liquid_sdk.cs

using breez_liquid_sdk.breez_liquid_sdk;

try
{
    var mnemonic = "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about";

    var config = BreezLiquidSdkMethods.DefaultConfig(Network.Testnet);

    var connectReq = new ConnectRequest(config, mnemonic);
    BindingLiquidSdk sdk = BreezLiquidSdkMethods.Connect(connectReq);

    GetInfoResponse? info = sdk.GetInfo();

    Console.WriteLine(info!.pubkey);
}
catch (Exception e)
{
    Console.WriteLine(e.Message);
    throw;
}
